src/entity/user/CommonUser.java

```diff
@@ -182,4 +184,9 @@ public void setPhone(String num) {
 		this.phone = num;
 	}
 
+	@Override
+	public List addNotif(List<String> notif) {
+		...
```

Comment:
so in addnotif the method should take a notification (string) instead of a list of notifications and add it to the notifications string (list). might make sense to do linkedlist too, but just declare as List like prof said. unless if u want to return the updated list(which u can), u can just make it void. up to u